CMakeLists.txt

```diff
@@ -93,8 +93,14 @@ endif ()
 
 set (OPENCL_ICD_LOADER_HEADERS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/inc" CACHE PATH "Path to OpenCL Headers")
 
-add_library (OpenCL ${OPENCL_ICD_LOADER_SOURCES})
-add_library (OpenCL::OpenCL ALIAS OpenCL)
+if (${OPENCL_ICD_LOADER_BUILD_SHARED_LIBS} OR ${BUILD_SHARED_LIBS})
+  add_library (OpenCL SHARED ${OPENCL_ICD_LOADER_SOURCES})
+  add_library (OpenCL::OpenCL ALIAS OpenCL)
+else()
+  add_library (OpenCL SHARED ${OPENCL_ICD_LOADER_SOURCES})
+  add_library (OpenCL::OpenCL ALIAS OpenCL)
+endif()
```

Comment:
`add_library (OpenCL::OpenCL ALIAS OpenCL)` need not be duplicated. It can go outside the conditional.

Reply:
Done